pkgs/tools/security/pynitrokey/default.nix

```diff
@@ -0,0 +1,59 @@
+{ python3Packages, lib }:
+
+with python3Packages;
+
+let
+  nkdfu = buildPythonPackage rec {
+    pname = "nkdfu";
+    version = "0.1";
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-Y8GonfCBi3BNMhZ99SN6/SDSa0+dbfPIMPoVzALwH5A=";
+    };
+
+    format = "flit";
+    propagatedBuildInputs = [
+      fire
+      tqdm
+      intelhex
+      libusb1
+    ];
+  };
```

Comment:
Can we move this to pythonPackages?

Reply:
Can I add 2 packages in 1 commit? or should create 2 `init at x.y.z` commits?

Reply:
Please do two commits. Can be all done in this PR.